src/analysis/ml.py

```diff
@@ -83,44 +80,70 @@ def pca(dataframe: pd.DataFrame, output_png: str, output_file: str, output_coord
     @param output_file: the filename to save the variance explained by each component
     @param output_coord: the filename to save the coordinates of each algorithm
     """
+
     df = dataframe.reset_index(drop=True)
+    column_names = df.head()
```

Comment:
Remove this line?

Reply:
Need that to get the column names, used as the labels in the graphs

Reply:
If found that confusing because `head` returns the first n rows (default 5). Can we use https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.columns.html to get them directly?